src/sentry/static/sentry/app/views/settings/projectPlugins/projectPluginRow.tsx

```diff
@@ -31,8 +40,8 @@ class ProjectPluginRow extends React.PureComponent {
     onChange(id, !enabled);
     trackIntegrationEvent(
       {
-        eventKey: `integrations.${!enabled ? 'enabled' : 'disabled'}`,
-        eventName: `Integrations: ${!enabled ? 'Enabled' : 'Disabled'}`,
+        eventKey: `integrations.${!enabled ? 'enabled' : 'disabled'}` as any,
+        eventName: `Integrations: ${!enabled ? 'Enabled' : 'Disabled'}` as any,
```

Comment:
Can we add `TODO(ts)` here or instead of `as any`, make it `as 'integrations.disabled'|'integrations.enabled'`

Reply:
@billyvg fixed